pkg/util/cluster/cluster.go

```diff
@@ -403,38 +430,33 @@ func (c *Cluster) createCluster(ctx context.Context, vnetResourceGroup, clusterN
 }
 
 func (c *Cluster) fixupNSGs(ctx context.Context, vnetResourceGroup, clusterName string) error {
-	timeoutCtx, cancel := context.WithTimeout(ctx, 10*time.Minute)
-	defer cancel()
+	rp := fmt.Sprintf("/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Network/virtualNetworks/dev-vnet", c.env.SubscriptionID(), c.env.ResourceGroup())
+	cluster := fmt.Sprintf("/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Network/virtualNetworks/dev-vnet", c.env.SubscriptionID(), vnetResourceGroup)
 
-	// very occasionally c.securitygroups.List returns an empty list in
-	// production.  No idea why.  Let's try retrying it...
-	var nsgs []mgmtnetwork.SecurityGroup
-	err := wait.PollImmediateUntil(10*time.Second, func() (bool, error) {
-		var err error
-		nsgs, err = c.securitygroups.List(ctx, "aro-"+clusterName)
-		return len(nsgs) > 0, err
-	}, timeoutCtx.Done())
-	if err != nil {
-		return err
+	clusterProp := &mgmtnetwork.VirtualNetworkPeeringPropertiesFormat{
+		RemoteVirtualNetwork: &mgmtnetwork.SubResource{
```

Comment:
I expect you will probably want this here and below too:
```
AllowVirtualNetworkAccess: to.BoolPtr(true),
AllowForwardedTraffic:     to.BoolPtr(true),
```

(see function halfPeering)

Reply:
done